src/hooks/_test_/usePokemonData.test.ts

```diff
@@ -0,0 +1,283 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { renderHook, act } from '@testing-library/react';
+import { usePokemonData, type UsePokemonDataConfig } from '../usePokemonData';
```

Comment:
Fix: Use consistent import paths with aliases
Mixed relative and alias imports create inconsistency. Changed to use aliases throughout for:

- Consistency: All imports follow the same pattern
- Maintainability: Easier refactoring when moving files
- Readability: Clear module organization without relative path complexity

```suggestion
import { usePokemonData, type UsePokemonDataConfig } from '@hooks/usePokemonData';
```